packages/contracts/src/utils/JwtVerifier.sol

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "../interfaces/IJwtGroth16Verifier.sol";
+import {UUPSUpgradeable} from "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import {strings} from "solidity-stringutils/src/strings.sol";
+import {IVerifier, EmailProof} from "../interfaces/IVerifier.sol";
+
+contract JwtVerifier is IVerifier, OwnableUpgradeable, UUPSUpgradeable {
+    using strings for *;
+
+    IJwtGroth16Verifier groth16Verifier;
+
+    uint256 public constant ISS_FIELDS = 2;
+    uint256 public constant ISS_BYTES = 32;
+    uint256 public constant COMMAND_FIELDS = 20;
+    uint256 public constant COMMAND_BYTES = 605;
+    uint256 public constant AZP_FIELDS = 3;
+    uint256 public constant AZP_BYTES = 72;
```

Comment:
I think it is ok to hardcode them because the groth16 verifier also assumes specific values of those variables.
It is troublesome for a deployer to specify the same values fixed by the circuit.